src/packages/@ncigdc/components/CartDownloadDropdown.js

```diff
@@ -60,31 +60,24 @@ const downloadCart = ({
   setState: Function,
 }) => {
   const { authorized, unauthorized } = authPartitionFiles({ user, files });
-  if (unauthorized.doc_count > 0) {
+  if (!user) {
```

Comment:
this should still check for `unauthorized.doc_count > 0`, reason being there are `files.access = open` files that anyone even annon can download